### PR TITLE
Skydoc website.

### PR DESCRIPTION
**Pull request by davidzchen**
screen shot 2016-07-30 at 4 07 20 pm

### CODE REVIEW

File: serve-docs.sh [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+$0 <target directory> [<serving prefix>]
+     Builds docs as static web pages in <target directory>.
+     Replaces absolute paths in the resulting HTML with <serving prefix>,
+     or, if it is not specified, with <target directory>.
+EOF
+  exit 0
+fi
+
+if [[ "${1-}" == [0-9]* ]]; then
+  readonly PORT=$1
```
Comment (damienmg) [inline serve-docs.sh]: nit: Here and everywhere below, quote $ expansions

File: serve-docs.sh [hunk 2]
```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
```
Comment (damienmg) [inline serve-docs.sh]: I guess this is forked from the Bazel repository?

File: site/contributing.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+[pull-request]: https://github.com/bazelbuild/skydoc/pulls
+[cla]: https://cla.developers.google.com/
+
+## Core Contributors
+
+The current group of Skydoc core contributors are:
+
+* [davidzchen](https://github.com/davidzchen)
+* [damienmg](https://github.com/damienmg)
+* [laurentlb](https://github.com/laurentlb)
```
Comment (damienmg) [inline site/contributing.md]: Maybe add @dslomov and @fweikert? (wait for them to say yes/no)

File: serve-docs.sh (hunk 2 above)
Comment (davidzchen) [inline serve-docs.sh]: Yes, it is.

File: serve-docs.sh (hunk 1 above)
Comment (davidzchen) [inline serve-docs.sh]: Done.

File: site/docs/faq.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: FAQ
+---
+
+## Is Skydoc used to generate the documentation on [Bazel.io](http://bazel.io)?
+
+The [Build Encyclopedia](http://bazel.io/docs/be/overview.html) documents the
```
Comment (damienmg) [inline site/docs/faq.md]: Add "Not for now,"

And also that we want to provide a comprehensive set dictionnary of available Skylark rule that will use skydoc.

**Comment by damienmg** [comment]
Ok I am finished reviewing it.

Just a few nits and overall LGTM.

Follow-up question:
  Should we create skydoc.bazel.io?
  Should we add it to the automatic push process (i.e. when you push to skydoc, it regenerates the site and push it to skydoc.bazel.io)?

File: site/docs/faq.md (hunk 4 above)
Comment (davidzchen) [inline site/docs/faq.md]: Done.

**Comment by davidzchen** [comment]
Thanks for reviewing!

> Should we create skydoc.bazel.io?
> Should we add it to the automatic push process (i.e. when you push to skydoc, it regenerates the site and push it to skydoc.bazel.io)?

Yes, please. :) That would be great. Once skydoc.bazel.io is up, I'll add a "Documenting Rules" link on the sidebar on bazel.io that links to skydoc.bazel.io.

@dslomov, @fweikert - I have added you to the list of core contributors for Skydoc (see `site/contributing.md`) per Damien's suggestion above. Let me know if you prefer not to be on the list.

File: site/contributing.md (hunk 3 above)
Comment (davidzchen) [inline site/contributing.md]: @dslomov, @fweikert - I have added you to the list of core contributors for Skydoc (see `site/contributing.md`) per Damien's suggestion above. Let me know if you prefer not to be on the list.

**Comment by fweikert** [comment]
David, thank you!

**Pull request merged**